packages/mint-components/src/components/sqm-task-card/sqm-task-card-view.tsx

```diff
@@ -0,0 +1,336 @@
+                </div>
+              )}
+              {showComplete && (
+                <span class={taskUnavailable ? "icon neutral" : "icon"}>
+                  {checkmark_circle}
+                </span>
+              )}
+              <span class={taskUnavailable ? "value" : "value black"}>
+                {props.rewardAmount}
+              </span>
+              <span class="text">{props.rewardUnit}</span>
+            </div>
+          )}
+        </div>
+
+        {props.loading ? (
+          <sl-skeleton
+            style={{ width: "42%", margin: "0 16px", height: "12px" }}
+          />
+        ) : (
+          <div class={taskUnavailable ? "title" : "title black"}>
+            {props.cardTitle}
+          </div>
+        )}
+        {props.loading ? (
+          <sl-skeleton style={{ margin: "12px 16px", height: "12px" }} />
+        ) : (
+          <Details {...props} />
+        )}
+        {props.showProgressBar && props.loading ? (
+          <sl-skeleton style={{ margin: "0 16px", height: "12px" }} />
+        ) : (
+          props.showProgressBar && (
+            <ProgressBarView
+              {...props}
+              complete={taskComplete}
+              expired={taskUnavailable}
+            />
+          )
+        )}
+        <div class={sheet.classes.Footer}>
+          {props.loading ? (
+            <sl-skeleton
+              style={{ width: "25%", marginLeft: "auto", height: "12px" }}
+            />
+          ) : (
+            <div style={{ display: "contents" }}>
+              <span class="text">
+                {props.repeatable && (
+                  <div>
+                    <span
+                      class={
+                        repetitions > 0
+                          ? taskUnavailable
+                            ? "icon neutral"
+                            : "icon success"
+                          : "icon"
+                      }
+                    >
+                      {arrow_left_right}
+                    </span>
+                    <span
+                      class={
+                        repetitions > 0
+                          ? taskUnavailable
+                            ? "neutral"
+                            : "success"
+                          : ""
+                      }
+                    >
+                      {"Completed " + repetitions + " times"}
+                    </span>
+                  </div>
+                )}
+                {props.showExpiry && !taskUnavailable && (
+                  <span>
+                    {"Ends " + dateEnd.toLocaleString(DateTime.DATE_FULL)}
+                  </span>
+                )}
+              </span>
+
+              <sl-button
+                class={
+                  taskComplete || taskUnavailable ? "action disabled" : "action"
+                }
+                type="primary"
+                size="small"
+                onClick={() => window.open(props.buttonLink)}
```

Comment:
Redirect the top level window instead of opening a new tab.

```suggestion
                onClick={() => window.top.location = props.buttonLink}
```